code/modules/martial_arts/judo.dm

```diff
@@ -0,0 +1,53 @@
+/datum/martial_art/judo
+	name = "Corporate Judo"
+	has_explaination_verb = TRUE
+	no_baton = TRUE
+	combos = list(/datum/martial_combo/judo/discombobulate, /datum/martial_combo/judo/eyepoke, /datum/martial_combo/judo/judothrow, /datum/martial_combo/judo/armbar, /datum/martial_combo/judo/fatailty)
+	var/armbar_active = FALSE //used to check if the victim is in an armbar, useful for finishing moves
+
+//Corporate Judo Belt
+
+/obj/item/storage/belt/champion/judo
+	name = "Corporate Judo Belt"
+	var/datum/martial_art/judo/style = new
```

Comment:
This is going to copy everything from its champion subtype. Unless you intend for it to actually store items in the belt, I recommend you change the path to `/obj/item/judobelt` and use `slot_flags = SLOT_BELT`. Youll need to copy some stuff like the icon_state from the champion belt, but it should be a fairly small change.

Reply:
whenever I do this, an error is thrown from line 12. Im unsure of the cause, but leaving it as a subpath of the champion belt seems to allow it to behave. In the latest commit I removed its storage features.

Reply:
Pathing issues resolved!